airflow/task/task_runner/standard_task_runner.py

```diff
@@ -73,6 +72,9 @@ def _start_by_fork(self):  # pylint: disable=inconsistent-return-statements
             # [1:] - remove "airflow" from the start of the command
             args = parser.parse_args(self._command[1:])
 
+            self.log.info('Running: %s', self._command)
+            self.log.info('Job %s: Subtask %s', self._task_instance.job_id, self._task_instance.task_id)
```

Comment:
Was this debugging or did you mean to leave it in?

Reply:
I meant to leave it in. Mainly to have similar log statements both when using "os.fork" and "subprocess"